cmd/lnmuxd/config.go

```diff
@@ -117,5 +120,18 @@ func loadConfig(filename string) (*Config, error) {
 		return nil, err
 	}
 
+	// Check if the user provides an environment variable for PG DSN and load it
+	loadStringEnvVariable(PostgresDSNEnvKey, &cfg.DB.DSN)
+
+	// Check if the user provides an environment variable for the identity key and load it
+	loadStringEnvVariable(IdentityKeyEnvKey, &cfg.IdentityKey)
+
 	return &cfg, nil
 }
+
+func loadStringEnvVariable(key string, value *string) {
```

Comment:
If we need to extend this behavior we can either create X functions for each type or using generics/interface.

For example:

```
func loadEnvVariableGen[T any](key string, value T) {
	if val, ok := os.LookupEnv(key); ok {
		log.Infof("Environment variable %q found", key)
		if reflect.ValueOf(value).Kind() != reflect.Ptr {
			log.Info("A pointer is needed for 'loadEnvVariableGen', environment variable discarded")
			return
		}

		switch t := any(value).(type) {
		case *string:
			*(any(value).(*string)) = val
                default
                	log.Infof("Don't know how to load environment variable in type %T", t)
		}
	}
}
```

Reply:
I like the simple version for now. Looks great!